templates/package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "build": "node build.js",
     "lint": "eslint modern/src && stylelint modern/src",
-    "test": "jest",
+    "test": "node --import tsx --test modern/src/helper.test.ts",
```

Comment:
Is there a way to run `*.test.ts`?

Reply:
> On windows environment.
It seems wildcard is not supported when running on Node 20.
https://github.com/nodejs/node/issues/50658

Ok I see.